core/src/repl/mod.rs

```diff
@@ -415,15 +421,18 @@ pub fn print_help(out: &mut impl Write, arg: Option<&str>) -> std::io::Result<()
                 )?;
             }
             Ok(c @ CommandType::Query) => {
-                writeln!(out, ":{c} <identifier> [field path]")?;
+                writeln!(out, ":{c} <field path>")?;
                 print_aliases(out, c)?;
                 writeln!(out, "Print the metadata attached to a field")?;
                 writeln!(
                     out,
-                    "<identifier> is valid Nickel identifier representing the record to look into."
+                    "<field path> is a dot-separated sequence of identifiers pointing to a field. \
+                    Fields can be quoted if they contain special characters, \
+                    very much as in normal Nickel source code.\n"
```

Comment:
```suggestion
                    just like in normal Nickel source code.\n"
```
This would sound better to me, I think.